src/main/java/seedu/address/storage/JsonAdaptedPerson.java

```diff
@@ -40,7 +40,7 @@ public JsonAdaptedPerson(@JsonProperty("name") String name, @JsonProperty("phone
         this.name = name;
         this.phone = phone;
         this.email = email;
-        this.address = address;
+        this.id = address;
```

Comment:
I think this should be ``` this.id = id``` and not ```this.id = address;```

This means that the Json constructor needs to be updated too to take in ```@JsonProperty("address") String id``` instead of ```@JsonProperty("address") String address```.